mbuild/tests/test_compound.py

```diff
@@ -869,9 +869,8 @@ def test_clone_outside_containment(self, ch2, ch3):
         with pytest.raises(MBuildError):
             ch3_clone = mb.clone(ch3)
 
-    def test_load_mol2_mdtraj(self):
-        with pytest.raises(KeyError):
-            mb.load(get_fn('benzene-nonelement.mol2'))
+    def test_load_nonelement_mol2(self):
+        mb.load(get_fn('benzene-nonelement.mol2'))
```

Comment:
Do we want to also test the case when the element could not be inferred from atom name?